.github/workflows/nvm.yml

```diff
@@ -5,9 +5,47 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: node --version
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - run: |
           nvm install --latest-npm --no-progress
-          echo "::set-env name=PATH::${NVM_BIN}:${PATH}"
+          echo "$(dirname $(which node))" >> $GITHUB_PATH
```

Comment:
[suggest] I'm unsure but is `nvm which` safer than just `which`?

```suggestion
          echo "$(dirname $(nvm which node))" >> $GITHUB_PATH
```

Reply:
Oh interesting, I didn't know `nvm which` was a command - happy to apply if this works

Reply:
so according to [the docs](https://github.com/nvm-sh/nvm#:~:text=You%20can%20also%20get%20the%20path%20to%20the%20executable%20to%20where%20it%20was%20installed%3A) it appears that we will need a Node.js version specifier to get a path:

> You can also get the path to the executable to where it was installed:
>
> ```
> nvm which 12.22
> ```

if we want to go this direction, we could try using `*` or something...?

Reply:
actually, looking closer at [the version aliases in the `nvm` docs](https://github.com/nvm-sh/nvm#:~:text=In%20place%20of,version%20of%20node), there is also a `node` version alias, weird!

> In place of a version pointer like "14.7" or "16.3" or "12.22.1", you can use the following special default aliases with `nvm install`, `nvm use`, `nvm run`, `nvm exec`, `nvm which`, etc:
>
>  - `node`: this installs the latest version of [`node`](https://nodejs.org/en/)

so I made the change in 3f4af212a6b31e2107db034df429371cb82365f2

Reply:
`nvm which` is proper because `.nvmrc` version is used expectedly:

https://github.com/ybiquitous/using-nvm-on-github-action/pull/3/checks#step:6:7

<img width="211" alt="image" src="https://github.com/user-attachments/assets/58d1ba1e-c996-4a2a-9e33-928757809536">

https://github.com/ybiquitous/using-nvm-on-github-action/blob/8548472d5e23e281c073ed854bae95a3e5c67e87/.nvmrc#L1

Reply:
Great, so we'll leave `nvm which node` as it currently is? (since it's working)

Or switch to `nvm which`?

Reply:
Let's leave `nvm which node` as-is 🙏🏼